wordpress/wp-content/plugins/memberful-wp/src/paywall/preview.php

```
@@ -0,0 +1,94 @@
+	 */
+	public static function handle(): void {
+		check_ajax_referer( self::NONCE_KEY, 'nonce' );
+
+		if ( ! current_user_can( 'manage_options' ) ) {
+			wp_send_json_error( array( 'message' => 'forbidden' ), 403 );
+		}
+
+		$raw = filter_input( INPUT_POST, 'config', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+		$raw = is_array( $raw ) ? $raw : array();
+
+		$config = Memberful_Paywall_Sanitizer::sanitize( $raw, Memberful_Paywall_Config::defaults() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Memberful_Paywall_Sanitizer' in method 'handle'. (undefined)

(StaticAccess)